docs/user-management-and-perms/index.rst

```diff
@@ -119,15 +116,14 @@ Support for AWS.
 ^^^^^^^^^^^^^^^^^^^^^^
 
 These permissions are Rackspace product-specific. This is where you manage
-Fanatical Support for AWS permissions. This guide doe not cover other
-product permissions.
+Product Access. This guide doe not cover other product permissions.
 
 There are three Fanatical Support for AWS permissions:
 
 * **Allow this user to add AWS Accounts**: Enables the user to add
   additional AWS accounts for Rackspace to manage. These could be new or
   existing AWS accounts.
-* **Fanatical Support for AWS**: Controls what access the user has within the
+* **Product Access**: Controls what access the user has within the
```

Comment:
I recently found out that we do not need the word Technology in Rackspace Technology Customer Portal.